src/actions/order-actions.js

```diff
@@ -66,18 +73,27 @@ export function convertCheckoutToOrder (cart, paymentMethod, order) {
       line_items_resources: prepareLineItems(cart),
       shipping_address: prepareShippingAddress(cart.shipping_address),
       shipping_method: prepareShippingMethod(cart.shipping_method),
-      discount_summaries: discountSummaries(cart, lineItems),
-      sub_total: cart.sub_total,
-      total: cart.total,
-      shipping_total: cart.shipping_total,
-      tax: cart.tax,
-      placed_at: new Date().toISOString()
+      discount_summaries: discountSummaries,
```

Comment:
I am not fond of these calculations but it is what the new order endpoint requires.